docs/content/guidelines/design.mdx

```diff
@@ -0,0 +1,300 @@
+        </Flex>
+        <Caption>Use depth when it adds meaning</Caption>
+    </Do>
+    <Dont>
+        <Flex bg="gray.1" justifyContent="center">
+            <img width="400" alt="" src="https://compai.pub/v1/png/bbe56e09aff2a5f8b0dfa7dbbc37f1cc4d6b4f464e4f1b06d0b78dc5cf970454"/>
+        </Flex>
+        <Caption>Don't use a 2D perspective if it makes the icon unrecognizable</Caption>
+    </Dont>
+</Grid>
+
+## Pixel alignment
+
+Align the outer edge of shapes to pixel boundaries when possible.
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={4}>
+    <Do>
+        <Flex bg="gray.1" justifyContent="center">
+            <img width="400" alt="" src="https://compai.pub/v1/png/7fb5cb03a4e6d1164ad31b1ed8d3538f00c69a788c2d3bfd8ad282398b517c04"/>
+        </Flex>
+        <Caption>Align the outer edge of shapes to pixel boundaries</Caption>
+    </Do>
+    <Dont>
+        <Flex bg="gray.1" justifyContent="center">
+            <img width="400" alt="" src="https://compai.pub/v1/png/39e340f63b3f8f6a8e86f3a656b53777caddfaec757ee98a0f64c8e7de245f7d"/>
+        </Flex>
+        <Caption>Don't align the inner edge of shapes to pixel boundaries</Caption>
+    </Dont>
+</Grid>
+
+
+## Arrowheads
+
+Use line arrowheads unless there is not enough room.
+
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={4}>
+    <div>
+        <Flex bg="gray.1" justifyContent="center">
+            <img width="400" alt="" src="https://compai.pub/v1/png/d8f70676d9e78c9bf5aab2961eb65d5aa311bf8119f11be229c8056838a2cd6e"/>
+        </Flex>
+        <Caption>Line arrowhead in sign-out icon</Caption>
+    </div>
+    <div>
+        <Flex bg="gray.1" justifyContent="center">
+            <img width="400" alt="" src="https://compai.pub/v1/png/603edbc609906bb955775f45d0d0257b25805ab309b1ad86111b029e3b57a0fc"/>
+        </Flex>
+        <Caption>Filled arrowhead in link-external icon</Caption>
+    </div>
+</Grid>
+
+
+## Contributing
+
+### Creating a pull request
+
+If you work at GitHub, you can use the [Octicons Push](https://www.figma.com/community/plugin/825432045044458754/Octicons-Push) Figma plugin to start an Octicons pull request from Figma.
+
+Here's how it works:
+1. Select the icon frames you want to commit. Make sure the frames are either 16x16 or 24x24 and that you've outlined all strokes.
+1. Open the Octicons Push plugin.
+1. Select the branch you want to commit to. You can choose an existing branch or create a new branch.
+1. Press "Commit." The plugin will then export, commit, and push the selected icons to the branch you chose. If you chose to create a new branch, the plugin will give you a link to where you can start a new pull request with your branch.
+
+
+![demo](https://user-images.githubusercontent.com/4608155/77948730-b1a24600-727a-11ea-9c39-040be9a12963.gif)
+
+### Review questions
```

Comment:
Will there be a sentence here like: _Before starting to design a new octicon, consider the following:_

Reply:
Good idea. I added a short intro for this section. How does this sound?

> Here are a few questions we'll ask when reviewing new octicons. Keep these in mind as you're designing: